src/components/SingleArticle.jsx

```diff
@@ -15,6 +16,26 @@ const SingleArticle = () => {
     }, []);
     function handleCommentsClick() {
         setShowComments((currShow)=>{return !currShow})
+    };
+    function handleVoteClick(voteType) {
+        let newVotes
```

Comment:
Great that you've got one flexible function that's able to handle up and down voting 👏